frontend/src/components/users/UsersSideBar.jsx

```diff
@@ -12,7 +12,7 @@ export default function SideBar(props) {
                     <NavLink className='nav-link d-block m-2' to={`/users/profile/${props.loggedUser.id}`} > Profile </NavLink>
                 </li>
                 <li className='nav-item'>
-                    <NavLink className='d-block m-2' to={`/users/${props.loggedUser.id}/wishlist`} > Wishlist </NavLink>
```

Comment:
I believe it's better to leave the route how it was
to={`/users/${props.loggedUser.id}/wishlist`} make more sense, at least to me (if you all agree on the other way (to={`/users/wishlist/${props.loggedUser.id}`}) then we can change it